Added note about PowerShell version (#3464)
MicrosoftDocs/OfficeDocs-SharePoint: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md

    > If you installed a previous version of the SharePoint Online Management Shell, go to Add or remove programs and uninstall "SharePoint Online Management Shell".
    > 
    > Make sure you review the System Requirements and Install Instructions. The app isn't supported on Mac.
  

>   > [!NOTE]
>     > Make sure version is 16.0.22810.12000 or lower. You might receive error Microsoft.SharePoint.Client.ResourceNotFoundException for higher versions (This is temporary note). 
>     > You can install specific version from Windows PowerShell using command Install-Module. Example: Install-Module -Name Microsoft.Online.SharePoint.PowerShell -RequiredVersion 16.0.22810.12000 

Reviewer: ```suggestion
    > [!NOTE]
    > Make sure the version is 16.0.22810.12000 or earlier. You might receive an error `Microsoft.SharePoint.Client.ResourceNotFoundException` for later versions. 
    >
    > You can install a specific version from Windows PowerShell using the `Install-Module` command, for example `Install-Module -Name Microsoft.Online.SharePoint.PowerShell -RequiredVersion 16.0.22810.12000`. 
```
